server/balancer_test.go

```diff
@@ -102,6 +102,14 @@ func (s *testBalancerSuite) updateStore(c *C, clusterInfo *clusterInfo, storeID
 	c.Assert(ok, IsTrue)
 }
 
+func (s *testBalancerSuite) updateStoreState(c *C, clusterInfo *clusterInfo, storeID uint64, state metapb.StoreState) {
+	storeInfo := clusterInfo.getStore(storeID)
+	storeInfo.store.State = state
+	clusterInfo.addStore(storeInfo.store)
```

Comment:
I mean updateStoreStatus can update cluster cache, so i think it is needless to call add store again.